hare4/hare_test.go

```diff
@@ -928,7 +928,9 @@ func TestHandler(t *testing.T) {
 	})
 }
 
-func gatx(id types.ATXID, epoch types.EpochID, smesher types.NodeID, base, height uint64) types.ActivationTx {
+const epoch = types.EpochID(1)
```

Comment:
Same as above, epoch 1 is different from all other epochs, we shouldn't only test for that specific. We should have at least a single test where epoch is a different value.